addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxCubeBridgeHandler.java

```diff
@@ -181,9 +189,19 @@ public void handleCommand(ChannelUID channelUID, Command command) {
     @Override
     public void dispose() {
         logger.debug("Handler disposed.");
-        stopAutomaticRefresh();
```

Comment:
This was caused by another binding. Since then I've had this code (except for the last commit, which removes some of the excess ReentrantLocks) running without any problems.